addons/web/static/src/js/libs/jSignatureCustom.js

```diff
@@ -0,0 +1,44 @@
+function _renderImageOnCanvas( data, formattype, rerendercallable ) {
+    'use strict'
+    // #1. Do NOT rely on this. No worky on IE 
+    //   (url max len + lack of base64 decoder + possibly other issues)
+    // #2. This does NOT affect what is captured as "signature" as far as vector data is 
+    // concerned. This is treated same as "signature line" - i.e. completely ignored
+    // the only time you see imported image data exported is if you export as image.
+
+    // we do NOT call rerendercallable here (unlike in other import plugins)
+    // because importing image does absolutely nothing to the underlying vector data storage
+    // This could be a way to "import" old signatures stored as images
+    // This could also be a way to import extra decor into signature area.
+    
```

Comment:
I understand that you copied the original comment of the library but I would add an Odoo comment to explain why this override is needed and what it does.

Reply:
Done :)